charts/nutanix-csi-storage/templates/service-prometheus-csi.yaml

```diff
@@ -6,14 +6,14 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: csi-metrics-service
+  name: nutanix-csi-metrics
   namespace: {{ .Release.Namespace }}
   labels:
-    app: csi-provisioner-ntnx-plugin
+    app: nutanix-csi-metrics
 spec:
   type: ClusterIP
   selector:
-    app: csi-provisioner-ntnx-plugin
+    app: nutanix-csi-controller
```

Comment:
Add monitoring for other services also like in : https://github.com/subodh01/helm/blob/master/charts/nutanix-csi-storage/templates/service-prometheus-csi.yaml

Reply:
done